### PR TITLE
fix (?) some initials in the notes

### PR DESCRIPTION
**Pull request by MichaelChirico**
not 100% sure on attribution, took a guess whenever the initials couldn't be mapped to the list

### CODE REVIEW

**Comment by hturner** [comment]
The correction for CW looks right. @jennybc, @MayaGans are these things you recognise saying in the meeting? (Hoping tagging means you get a notification...)

**Comment by MayaGans** [comment]
I don't think I said the comment about only learning from experts but honestly don't remember 🤷‍♀️

**Comment by jennybc** [comment]
I did not say this:

> - (JB) One thing that is good to know/informative fact: There are companies that pay for people to contribute and become core developers. They do this as full-time work. It is part of the company goal. I’m only aware of big companies that do this.

File: team_minutes/2020-07-24.md [hunk 1]
```diff
@@ -221,7 +221,7 @@ R Consortium Diversity & Inclusion Group: Samantha Toet, gwynn sturdevant, Hadle
         - (CW) Excellent points, Robert. +1 from YB and NT.
         - (NT) Co-learning with peers can be better than learning from mentors. +1 from AM
         - (AM) I think it’s incorrect to say that someone who is not an expert is a worse mentor. Research shows that people closest to the novice state are often the best teachers.
-        - (GM) Sorry - I did not mean that one can only learn from an expert 
+        - (MG) Sorry - I did not mean that one can only learn from an expert
```
Comment (MayaGans) [inline team_minutes/2020-07-24.md:224]: I don't think this was me...

**Comment by emdodwell** [comment]
That comment about companies paying for people to contribute was Gabriela de Queiroz -- it looks like there was a typo in my transciption of (GB) so it must have been edited to JB instead of GQ. Apologies for the confusion.

File: team_minutes/2020-07-24.md (hunk 1 above)
Comment (MayaGans) [reply team_minutes/2020-07-24.md:224]: Could this `GM` also be `GB` as mentioned below?

Comment (emdodwell) [reply team_minutes/2020-07-24.md:224]: @MayaGans This was Robert Gentleman (RG), a comment copied out of chat.

**Comment by emdodwell** [comment]
@jennybc @hturner That comment about companies paying for people to contribute was Gabriela de Queiroz -- it looks like there was a typo in my transciption of (GB) so it must have been edited to JB instead of GQ. Apologies for the confusion.

The CW correction is correct.

File: team_minutes/2020-07-24.md [hunk 2]
```diff
@@ -234,7 +234,7 @@ R Consortium Diversity & Inclusion Group: Samantha Toet, gwynn sturdevant, Hadle
         - Groups formed
         - September to assess which issues have volunteers and projects
         - Feel free to get in touch with comments later
-    - Many thanks to HT for organizing from ST, AM, JB, EC, AV, RB, SKB, MCR, NT, GR, GQ, YB, CW, MG, ER
+    - Many thanks to HT for organizing from ST, AM, JB, EC, AV, RG, SKB, MCR, NT, GR, GQ, YB, CW, MG, ER
```
Comment (MichaelChirico) [inline team_minutes/2020-07-24.md:237]: thanks @emdodwell for the corrections above, any idea on this one?

Comment (emdodwell) [reply team_minutes/2020-07-24.md:237]: RG is the right correction. My goodness, I was exhausted when I incorporated the chat into the minutes and apparently completely forgot how the alphabet works -- I can do a quick spot check in its entirety tomorrow. Thanks for catching these, @MichaelChirico.

Comment (MichaelChirico) [reply team_minutes/2020-07-24.md:237]: i promise it's better than i'd do 😄 thanks again for transcribing

**Pull request merged**